corehq/apps/export/models/new.py

```diff
@@ -1238,14 +1268,16 @@ class SplitUserDefinedExportColumn(ExportColumn):
     )
     user_defined_options = ListProperty()
 
-    def get_value(self, doc, base_path, transform_dates=False, **kwargs):
+    def get_value(self, domain, doc_id, doc, base_path, transform_dates=False, **kwargs):
```

Comment:
Really? That seems a bit odd and also contrary to what you have here: https://github.com/dimagi/commcare-hq/pull/11613/files#diff-9087195bf99fb8ba2fda775b0556aceaR315